mmv1/third_party/terraform/resources/resource_compute_instance_template.go.erb

```diff
@@ -869,7 +869,8 @@ func resourceComputeInstanceTemplateCreate(d *schema.ResourceData, meta interfac
 		return err
 	}
 
-	networks, err := expandNetworkInterfaces(d, config)
+	// expandNetworkInterfaces2 should be replaced by expandNetworkInterfaces once gVNIC is GA for resource_compute_instance_template 
```

Comment:
```suggestion
	// expandComputeInstanceTemplateNetworkInterfaces should be replaced by expandNetworkInterfaces once gVNIC is GA for resource_compute_instance_template 
```

Let's use this naming pattern instead of appending a 2- we're not sure how long we'll keep these variants around for.